Homework2.java

```diff
@@ -0,0 +1,31 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+/**
+ * @autor Created by mariavasilenko on 12/3/18.
+ */
+public class Homework2 {
+
+    public static void main(String[] args) {
+        Scanner s = new Scanner(System.in);
+        int[] myArray = new int[10];
+        int i = 0;
+        while (i < myArray.length) {
```

Comment:
Better to use foreach construction to iterate through array but in your case it looks good